ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -533,6 +546,139 @@ module Intel = struct
     List.map (find_or_create ~__context) types
 end
 
+module MxGPU = struct
+  let amd_vendor_id = 0x1002
+
+  let ( *** ) = Int64.mul
+  let ( /// ) = Int64.div
+  let ( +++ ) = Int64.add
+  let ( --- ) = Int64.sub
+  let mib x = List.fold_left Int64.mul x [1024L; 1024L]
```

Comment:
This whole module is just a copy of the Intel module with minimal changes.
I wanted to keep the changes as small as possible so that it will be easy to refactor this two modules to be based on a shared functor.
That refactoring will be done in a future pull request.